core/object/make_virtuals.py

```diff
@@ -211,17 +219,18 @@ def run(target, source, env):
 
 #include "core/object/script_instance.h"
 
+inline constexpr uintptr_t INVALID_GDVIRTUAL_FUNC_ADDR = static_cast<uintptr_t>(-1);
```

Comment:
Given that this is used entirely internally by the other macros, I think we should prefix this with a leading underscore:

```suggestion
inline constexpr uintptr_t _INVALID_GDVIRTUAL_FUNC_ADDR = static_cast<uintptr_t>(-1);
```